djangoevents/utils.py

```diff
@@ -0,0 +1,34 @@
+import inspect
+from .domain import BaseEntity, BaseAggregate, DomainEvent
+
+
+def _list_subclasses(cls):
+    """
+    Recursively lists all subclasses of `cls`.
+    """
+    return cls.__subclasses__() + [g for s in cls.__subclasses__() for g in _list_subclasses(s)]
```

Comment:
This is first time I see double list comprehension, and honestly am not sure if it's super readable ;-)